emulator.py

```diff
@@ -886,6 +886,61 @@ def delete_resumable_upload(bucket_name):
     db.delete_upload(upload_id, None)
     return flask.make_response("", 499, {"content-length": 0})
 
+# PROJECT ROUTES #
+
+@gcs.route("/projects/<project_id>/hmacKeys", methods=["POST"])
```

Comment:
Seems like this duplicates the code in:

https://github.com/googleapis/storage-testbench/blob/dfaed891836b1ef8152c79133dc3f5187ec8c954/gcs/project.py#L194-L287

We can either refactor the  `retry_test()` wrapper so it is usable from there, or we can move the functions from that location to here.

Reply:
BTW, if you decide to keep the functions in `projects.py`, then the existing unit tests will work.  Otherwise, please remember to change those unit tests to target the new functions.

Reply:
Oh sweet that makes sense, will work to integrate those then.